src/ontology/fbbt-edit.obo

```diff
@@ -126355,58 +126425,62 @@ creation_date: 2010-02-17T03:56:47Z
+name: sugar-sensing neuron of taste bristle
+def: "A sugar-sensing neuron that innervates a taste bristle (Montell, 2009). These neurons may also respond to amino acids (Dunipace, et al., 2001, Ganguly et al., 2017)." [FlyBase:FBrf0137203, FlyBase:FBrf0179508, FlyBase:FBrf0208984, FlyBase:FBrf0234535]
 comment: The projection of S cells (sugar-sensitive) of the labellum to the subesophageal zone, and of leg gustatory sensilla to the thoracic ganglion are segregated from the projections of L2 GRNs (bitter-sensitive) to the central brain (Wang et al., 2004).
-synonym: "category IS neuron" BROAD [FlyBase:FBrf0225914]
 synonym: "Gr5a neuron" NARROW [FlyBase:FBrf0234535]
+synonym: "S cell" EXACT [FlyBase:FBrf0208984]
 synonym: "S GRN" EXACT []
 synonym: "sugar neuron" EXACT [FlyBase:FBrf0137203]
 xref: FlyBrain_NDB:10284
-xref: FlyBrain_NDB:10285
 xref: FlyBrain_NDB:10382
 xref: VFB:FBbt_00100041
 intersection_of: FBbt:00005106 ! neuron
 intersection_of: capable_of GO:0001582 ! detection of chemical stimulus involved in sensory perception of sweet taste
-intersection_of: has_sensory_dendrite_in FBbt:00004162 ! labellar taste bristle
+intersection_of: has_sensory_dendrite_in FBbt:00005178 ! taste bristle
 created_by: sr544
 creation_date: 2010-03-09T11:09:32Z
 
 [Term]
 id: FBbt:00100042
-name: W cell
-def: "Gustatory receptor neuron of the short or long sensilla of the labellum that is responsive to water (Montell, 2009)." [FlyBase:FBrf0208984]
-synonym: "W GRN" EXACT []
-synonym: "water neuron" EXACT [FlyBase:FBrf0137203]
+name: water-sensing neuron of the labellum
+def: "A water-sensing neuron that innervates a short or long sensillum of the labellum (Montell, 2009)." [FlyBase:FBrf0208984]
+synonym: "W cell" BROAD [FlyBase:FBrf0208984]
 xref: VFB:FBbt_00100042
 is_a: FBbt:00005106 ! neuron
+intersection_of: FBbt:00058110 ! water-sensing neuron
+intersection_of: has_sensory_dendrite_in FBbt:00004162 ! labellar taste bristle
 relationship: capable_of GO:0050912 ! detection of chemical stimulus involved in sensory perception of taste
 relationship: has_sensory_dendrite_in FBbt:00004162 ! labellar taste bristle
 created_by: sr544
 creation_date: 2010-03-09T11:23:31Z
 
 [Term]
 id: FBbt:00100043
-name: L1 GRN
-def: "Gustatory receptor neuron of the short or long sensilla of the labellum that is responsive to low concentrations of salt (Montell, 2009)." [FlyBase:FBrf0208984]
-synonym: "classical salt neuron" RELATED [FlyBase:FBrf0137203]
+name: low salt-sensing neuron of the labellum
+def: "A low salt-sensing neuron that innervates a short or long sensillum of the labellum (Montell, 2009)." [FlyBase:FBrf0208984]
```

Comment:
looks like it is only long sensilla based on FBrf0229608, so this can be updated